File: Microcontroladores-2020-2/Pratica02/programa02.c
#include <stdio.h>

int main(void) {
  float A,B,X;
  printf("Informe um valor para B e para A:");
  scanf("%f%f", &B, &A);
  X = -B/A;
  printf("Valor de X: %.2f\n", X);
  return 0;
}

File: LTP2-2020-2/Pratica04/programa02.py
#Lista vazia com as temperaturas
# temperaturas = [23, 5, -6, 34]
temperaturas = []

print(temperaturas)

#Pede para o usuário digitar 5 temperaturas
contador = 0
while contador < 5:
  temperatura = float(input('Informe um temperatura:'))
  #Adiciona temperatura no final
  temperaturas.append(temperatura)
  #Só para ver o comportamento da lista entre as interações
  print(temperaturas)
  contador += 1 #contador = contador + 1

#Encontrar o maior Valor
maior = max(temperaturas)
print('Maior valor:', maior)

#Encontra o menor valor
menor = min(temperaturas)
print('Menor valor:', menor)
